docs/_data/translations.yml

```diff
@@ -3,11 +3,6 @@
   description: Bootstrap 中文文档
   url: http://v4.bootcss.com/
 
-- name: Chinese
-  code: zh
-  description: Bootstrap 4 中文文档教程
-  url: http://boot4.com/
```

Comment:
This link isn't broken, although it does redirect to `http://wiki.jikexueyuan.com/project/bootstrap4/`.